ui/public/index.html

```diff
@@ -3,6 +3,30 @@
   <head>
     <meta charset="utf-8" />
     <script src="%PUBLIC_URL%/lockdown.umd.js"></script>
+    <script>
+      // Allow the React dev environment to extend the console for debugging
+      // features.
+      const consoleTaming = '%NODE_ENV%' === 'development' ? 'unsafe' : 'safe';
+
+      const { pow: mathPow } = Math;
+      Math.pow = (base, exp) =>
+        typeof base === 'bigint' && typeof exp === 'bigint'
+          ? base ** exp
+          : mathPow(base, exp);
+
+      lockdown({
```

Comment:
@michaelfig I think I wasn't paying enough attention when we moved this stuff to a `lockdown.js` file. @samsiegart seems to be moving back; does that seem OK to you?